waspc/src/Wasp/Generator/DbGenerator/Common.hs

```diff
@@ -60,3 +66,42 @@ dbSchemaChecksumOnLastGenerateFileInDbRootDir = [relfile|schema.prisma.wasp-gene
 
 dbSchemaChecksumOnLastGenerateFileProjectRootDir :: Path' (Rel ProjectRootDir) (File DbSchemaChecksumOnLastGenerateFile)
 dbSchemaChecksumOnLastGenerateFileProjectRootDir = dbRootDirInProjectRootDir </> dbSchemaChecksumOnLastGenerateFileInDbRootDir
+
+data MigrateArgs = MigrateArgs
+  { _migrationName :: Maybe String,
+    _isCreateOnlyMigration :: Bool
+  }
+  deriving (Show, Eq)
+
+emptyMigrateArgs :: MigrateArgs
+emptyMigrateArgs = MigrateArgs {_migrationName = Nothing, _isCreateOnlyMigration = False}
+
+asArgs :: MigrateArgs -> [String]
+asArgs migrateArgs = do
+  concat . concat $
+    [ [["--create-only"] | _isCreateOnlyMigration migrateArgs],
+      [["--name", name] | Just name <- [_migrationName migrateArgs]]
+    ]
+
+parseMigrateArgs :: Maybe [String] -> MigrateArgs
+parseMigrateArgs Nothing = emptyMigrateArgs
+parseMigrateArgs (Just migrateArgs) = do
+  go migrateArgs emptyMigrateArgs
+  where
+    go :: [String] -> MigrateArgs -> MigrateArgs
+    go ("--create-only" : rest) mArgs = go rest mArgs {_isCreateOnlyMigration = True}
+    go ("--name" : name : rest) mArgs = go rest mArgs {_migrationName = Just name}
+    go _ mArgs = mArgs
```

Comment:
Right now I just ignore and bail for any unexpected arg. Should this `error` instead?

Reply:
I think error would be better! That way, if they make minor typo, they will know they did a typo, instead of thinking args was applied.

Reply:
Thanks, agree. I was leaning that way but this was the lazy "should I _realllyyy_ do it?" :D Will do!

Reply:
hehe sounds good :D!